php/libraries/Permission.class.inc

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * This file contains the Permission class.
+ *
+ * PHP Version 5-7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Rida Abou-Haidar <rida.loris@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+/**
+ * The Loris Permission class
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Rida Abou-Haidar <rida.loris@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Permission
+{
```

Comment:
As the class name is Permission, I would remove "permission" from the function name of the class

getPermissions             => getList
getPermissionLabels   => getLabels
getPermissionRoleIDs => getRoleIDs

Permission::GetLabels is in my sense better than Permission::GetPermissionLabels.

Same goes for Role class below

Reply:
not a bigfan of `getList` I don't think it's explicit enough but the rest makes sense.

what about `getName`

Reply:
getName**s**

Reply:
yeah, thats what I meant